awesome_cart/hooks.py

```diff
@@ -61,6 +61,8 @@
 	}
 }
 
+on_calculate_taxes_and_totals = "awesome_cart.utils.on_calculate_taxes_and_totals"
```

Comment:
I'm assuming you've tested this hook, cause I'm not sure if it *should* be in list if you're iterating through it in the code.

Reply:
ye, frappe combines all strings into an array. tested it and it works